test/CodeUnloading/CGlobalDecl.C

```diff
@@ -0,0 +1,43 @@
+//------------------------------------------------------------------------------
```

Comment:
Could you squash the test into the previous commit. It is better when browsing the history.